internal/output/diff.go

```diff
@@ -188,11 +188,18 @@ func tableForDiff(out Root, opts Options) string {
 			continue
 		}
 
+		var usageCost string
+		if out.Metadata.UsageApiEnabled {
```

Comment:
I'm worried that this condition is not sufficient because usage costs can also come from the usage file.  Essentially, we only want to show the "-" when `usageCostsMessage` is the "Usage costs CAN be estimated..." one.  It looks like that works out to showing - when `!out.Metadata.UsageApiEnabled && out.Metadata.UsageFilePath == "" && !out.Metadata.ConfigFileHasUsageFile`

as a sanity check in the API comment I also checked to make sure that both usage costs were zero or not present (I was worried about putting in a "-" and then having the total change not equal baseline change.